cmd/katapult/networks.go

```diff
@@ -34,20 +42,21 @@ func networksCmd(client *katapult.Client) *cobra.Command {
 				ref = core.OrganizationRef{SubDomain: subdomain}
 			}
 
-			nets, vnets, _, err := core.NewNetworksClient(client).List(cmd.Context(), ref)
+			nets, vnets, _, err := client.List(cmd.Context(), ref)
 			if err != nil {
 				return err
 			}
 
-			fmt.Println("Networks:")
+			stdout := cmd.OutOrStdout()
```

Comment:
Nitpick: As the `stdout` variable here may not always be actual STDOUT, I think calling this variable `out` or just `w` (writer) might cause slightly less confusion.